### ISSUE TITLE
Frameworks detected: Android framework is detected in the project Configure

### ISSUE THREAD

**Issue by adetheprince** [vkdinventor/MyVlcPlayer#1]
Hi there
I have seen your post in Stack Overflow and wanted to test this as I am still beginner.
I was hoping this example to work in Android Studio but I got that error,
Possible if you can tell us how can we bring this to AS?
Many thanks and love your work
PS, I have tested the other Vlc Example here and it never gave me that error with AS.

**Comment by vkdinventor** [comment]
Hi @adetheprince 
I am sorry for late reply. Please see updated repository. I have added all file  required to open this project in android Studio

**Comment by adetheprince** [comment]
Hi there
Thanks for the response, yes the app now is opening on Android Studio, but unfortunately no video is playing or it takes long and then I only get sound, the app is supposed to open up the video just as other vlc apps, I tested the same video links which works on vlc PC (ts), the demo link (mp4) is broken.
Can you please check or could this be because of ARM 7 Android I am using?
I tested this on 4.4 android box.

**Comment by adetheprince** [comment]
This app is similar to yours but not up to date, its working fine, can you look at this example?
https://github.com/gareoke/VLCPlayer/tree/master/VLCPlayerSample

**Comment by vkdinventor** [comment]
Hello @adetheprince 
 I have tested my app on android lolipop and it works fine. One more thing in my app I am streaming video online So can you please check if you are connected to internet? If you are connected to internet and still no viewo is playing send me log.

**Comment by adetheprince** [comment]
Hi there
Thanks for the quick reply, as I am sure the internet is OK since the sound is coming, 
I tested this app with this link (its a live channel)
http://149.202.77.226:7500/live/test02/test02/852.ts
can you test it your self please?
Many thanks.

**Comment by adetheprince** [comment]
using other apps I've made they work but poorer quality.
I need to test your vlc android.
I test exoplayer, Excomedia, and other VLC I found in here and all works but with issues.
The only stable i could managed is Exomedia after editing many codes but still have issues with quality (color) changed after few min , I hope VLC dont have this symptoms.
Possible if you could look at the original MrMaffen vlc android sdk in here and add your example?
https://github.com/mrmaffen/vlc-android-sdk
sorry for this but I'am new to Android in general.
Thanks.

**Comment by adetheprince** [comment]
The link in the demo is not working

https://archive.org/download/Popeye_forPresident/Popeye_forPresident_512kb.mp4
are you sure you tested recently?

**Comment by vkdinventor** [comment]
yeah I am sure Its working fine I just checked it again to make it sure. Just open the above link in your web browser if you are not able to play there may be firewall issue in your internet connection.

**Comment by adetheprince** [comment]
Hi there
Really strange the link is downloadable when clicking on here and the video works, but copy and pasting this to VLC (windows 10 ) its showing broken, never the less the link its not the issue here, but did you tested with the link I've sent you using your app?
Many thanks again, and sorry for too many questions.

**Comment by vkdinventor** [comment]
Hello 
http://149.202.77.226:7500/live/test02/test02/852.ts  this url is not working in my app, 
One more thing my app is supported only on armeabi-v7a devices.

**Comment by vkdinventor** [comment]
hi 
try exoplayer2 https://github.com/vkdinventor/MyGoogleExoplayer . 
I tested your url on MyGoogleExoplayer application and its working pretty fine. here is the apk
https://github.com/vkdinventor/MyGoogleExoplayer/blob/master/exoplayer.apk

**Comment by adetheprince** [comment]
Hi there
Many thanks for your reply again and sorry for the late response.
Yes I did try Exoplayer and Its OK but would also love to try VLC on different video format.
I really need your help on this please if you can bro.
I have download this lib [https://github.com/tianshaokai/vlc-android](url) and its working perfect so I wanted to make a play list that its Expandable List View like the Exoplayer (SampleChooserActivity) .
I have copied that to my project and after modifying/importing too many files I have managed to load the play list that is in json file  ("asset:///media.exolist.json",) so the file is loaded and I can see the list just as I wanted, but problem is the video is not playing, I understand I need to make few changes as the lib vlc is using MediaUtils.openStream(this, "http://mvvideo1.meitudata.com/579f07913f4254431.mp4", "唐唐讲段子");  if you look, this is opening a single stream but Exoplayer use different I think VideoPlayer.class after it load the list.
 I dont how to set on click listener in my SampleChooserActivity similar to the exoplayer one.
Can you please help me?

PS. according to the vlc android lib dev said you need to set onclick listener to the child on the ExpandableListView like this:

MediaUtils.openStream(context, url, "title");) method ExpandableListView onSampleSelected SampleChooserActivity but I am not sure where to add it in my code.

here is my code (SampleChooserActivity )

Thanks.

**Comment by adetheprince** [comment]
Full example

[https://github.com/google/ExoPlayer/blob/release-v2/demo/src/main/java/com/google/android/exoplayer2/demo/SampleChooserActivity.java](url)

**Comment by vkdinventor** [comment]
For now forget exoplayer-sample please go through below link. This explain how to implement expandable listview and modifiy its list with your urls.
http://www.journaldev.com/9942/android-expandablelistview-example-tutorial 

Now in onChildclick method get url and play it in vlc player.

**Comment by adetheprince** [comment]
Hi there

Many thanks for your reply bro.
I will look at that example and see how it goes.
I am only using the exoplayer example for Expandable List View as you see.
But I will now change and try the example you sent me.
Thanks you fro your help you are a star.

**Comment by adetheprince** [comment]
problem the guy who made the vlc-android play the video like this.
MediaUtils.openStream(this, "http://mvvideo1.meitudata.com/579f07913f4254431.mp4", "唐唐讲段子");
If its Listview stored in json file. How would I go from there?
Thanks.

**Comment by adetheprince** [comment]
Ok done that example and its working but then how to set the onclick listener to play the video
This example its showing a Toast.makeText OchildClick so how can I play the video?
MediaUtils.openStream(this);
I need to load this onclick as its only the code play video with that Vlc android from that guy, but how?

Here is the example you sent me can you edit to play video please?.
Thanks.

expandableListView.setOnChildClickListener(new ExpandableListView.OnChildClickListener() {
            @Override
            public boolean onChildClick(ExpandableListView parent, View v,
                                        int groupPosition, int childPosition, long id) {
                Toast.makeText(
                        getApplicationContext(),
                        expandableListTitle.get(groupPosition)
                                + " -> "
                                + expandableListDetail.get(
                                expandableListTitle.get(groupPosition)).get(
                                childPosition), Toast.LENGTH_SHORT
                ).show();
                return false;
            }
        });

**Comment by vkdinventor** [comment]
I am assuming that you have added url in expandableList as below
```
List<String> basketball = new ArrayList<String>();
        basketball.add(firsturl);
        basketball.add(secondurl);
```
     
here is on chlidclick implementaton.

```
public boolean onChildClick(ExpandableListView parent, View v,
int groupPosition, int childPosition, long id) {
String videoUrl= expandableListDetail.get(expandableListTitle.get(groupPosition)).get(childPosition);
	MediaUtils.openStream(this, videoUrl, "唐唐讲段子");
return false;
}
```

**Comment by vkdinventor** [comment]
closed #1

**Comment by vkdinventor** [comment]
Hope I helped you.

**Comment by adetheprince** [comment]
Hi there
Yes many thanks for your help and i am almost there!
So far I have used the Expandable List View and happy with it.
Many thanks for your help,.
As I have also asked the script owner too for help and got everything almost sorted, but one thing is I am straggling to do is how can I make the video to play on create and ask android to select any video from the list and play ( Auto Start Play), Basically I want my app to play the last viewed video or any number from the list (if its hard to store the last played video on cache) when it first load.
I am not sure which file to edit on that script, but I tried to ask the Manufist file to load VideoPlayActivity and its load the player but of course no video is selected so it wont play. Here is the link again [https://github.com/tianshaokai/vlc-android](url)
Again many thanks for your help bro.
